<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1549377841736" stop="1549377888462">
  <name>Test_Travelocity</name>
  <labels/>
  <test-cases>
    <test-case start="1549377841736" status="passed" stop="1549377862206">
      <description>Open the home page</description>
      <name>Travelocity.test_step_1</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="3252-MainThread"/>
        <label name="host" value="Dudnikov"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps>
        <step start="1549377852794" status="passed" stop="1549377860178">
          <name>Проверка корректного открытия сайта</name>
          <title>Проверка корректного открытия сайта</title>
          <attachments/>
          <steps/>
        </step>
      </steps>
    </test-case>
    <test-case start="1549377862224" status="passed" stop="1549377869255">
      <description>Filling the field Going to</description>
      <name>Travelocity.test_step_2</name>
      <attachments>
        <attachment source="4a7eb940-a1e2-4563-8722-ad70dd6d1537-attachment.txt" title="Captured log call" type="text/plain"/>
      </attachments>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="3252-MainThread"/>
        <label name="host" value="Dudnikov"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps>
        <step start="1549377862226" status="passed" stop="1549377867234">
          <name>Заполняем поле Going to</name>
          <title>Заполняем поле Going to</title>
          <attachments/>
          <steps/>
        </step>
      </steps>
    </test-case>
    <test-case start="1549377869259" status="passed" stop="1549377871498">
      <description>Filling of the check-in data</description>
      <name>Travelocity.test_step_3</name>
      <attachments>
        <attachment source="fdd7b1d6-85e7-41e5-96e6-784eda7a0693-attachment.txt" title="Captured log call" type="text/plain"/>
      </attachments>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="3252-MainThread"/>
        <label name="host" value="Dudnikov"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps>
        <step start="1549377869261" status="passed" stop="1549377869498">
          <name>Заполняем поле даты Check-In</name>
          <title>Заполняем поле даты Check-In</title>
          <attachments/>
          <steps/>
        </step>
      </steps>
    </test-case>
    <test-case start="1549377871502" status="passed" stop="1549377874511">
      <description>Filling of the check-out data</description>
      <name>Travelocity.test_step_4</name>
      <attachments>
        <attachment source="e714e0cc-9d22-4ed8-8545-f04293bf921f-attachment.txt" title="Captured log call" type="text/plain"/>
      </attachments>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="3252-MainThread"/>
        <label name="host" value="Dudnikov"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps>
        <step start="1549377871506" status="passed" stop="1549377872510">
          <name>Заполняем поле даты Check-Out</name>
          <title>Заполняем поле даты Check-Out</title>
          <attachments/>
          <steps/>
        </step>
      </steps>
    </test-case>
    <test-case start="1549377874515" status="failed" stop="1549377888462">
      <description>Choose and reserved a hotel</description>
      <name>Travelocity.test_step_5</name>
      <failure>
        <message>TimeoutException: Message: 
</message>
        <stack-trace>self = &lt;Test_Travelocity.Travelocity testMethod=test_step_5&gt;

    @pytest.allure.step("Выбираем отель и резервируем его")
    def test_step_5(self):
        """Choose and reserved a hotel"""
        TF.click_src_butt(self.Driver_Ef)
        TF.choose_hotel(self.Driver_Ef, self.data["hotelNum"])
&gt;       TF.reserved_hotel(self.Driver_Ef)

Test_Travelocity.py:56: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
Test_Func.py:59: in reserved_hotel
    until(EC.element_to_be_clickable(Loc.Common.Reserve)).click()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = &lt;selenium.webdriver.support.wait.WebDriverWait (session="bbf35fee738ea50245a20dcb79eaf80f")&gt;
method = &lt;selenium.webdriver.support.expected_conditions.element_to_be_clickable object at 0x03C49930&gt;
message = ''

    def until(self, method, message=''):
        """Calls the method provided with the driver as an argument until the \
            return value is not False."""
        screen = None
        stacktrace = None
    
        end_time = time.time() + self._timeout
        while True:
            try:
                value = method(self._driver)
                if value:
                    return value
            except self._ignored_exceptions as exc:
                screen = getattr(exc, 'screen', None)
                stacktrace = getattr(exc, 'stacktrace', None)
            time.sleep(self._poll)
            if time.time() &gt; end_time:
                break
&gt;       raise TimeoutException(message, screen, stacktrace)
E       selenium.common.exceptions.TimeoutException: Message:

C:\Users\dudnikov\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\support\wait.py:80: TimeoutException</stack-trace>
      </failure>
      <attachments>
        <attachment source="9e1e804e-04e0-4eb7-ac08-14479259ebb2-attachment.png" title="screenshot" type="image/png"/>
        <attachment source="0d69d43c-d0ad-4ab5-8f34-3b93670f3ba5-attachment.txt" title="Captured log call" type="text/plain"/>
      </attachments>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="3252-MainThread"/>
        <label name="host" value="Dudnikov"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps>
        <step start="1549377874517" status="failed" stop="1549377885217">
          <name>Выбираем отель и резервируем его</name>
          <title>Выбираем отель и резервируем его</title>
          <attachments/>
          <steps/>
        </step>
      </steps>
    </test-case>
  </test-cases>
</ns0:test-suite>
